packages/utilities/psammead-assets/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.2.5 | [PR#2115](https://github.com/bbc/psammead/pull/2115) Added media icons SVGs |
```

Comment:
```suggestion
| 2.3.0 | [PR#2115](https://github.com/bbc/psammead/pull/2115) Added media icons SVGs |
```
I think this is a minor change. Otherwise LGTM :+1: